specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/rbac.json

```diff
@@ -0,0 +1,544 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "requestBody",
+            "in": "body",
+            "required": true,
+            "description": "The properties required to create a new role definition.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionProperties"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The Role Definition create or update request was accepted and will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/SqlRoleDefinitionResponse"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidRoleDefinitionId - The Role Definition ID '{0}' is not valid. Role Definition ID must be a GUID.\n\n * 400 RoleDefinitionNameNullOrEmpty - A Role Definition must have a non-empty, unique name.\n\n * 400 InvalidRoleDefinitionType - The Role Definition type value '{0}' is not supported. The supported values are 'CustomRole'.\n\n * 400 MissingAssignableScopes - A Role Definition must have at least one assignable scope and all assignable scopes must be neither null nor empty.\n\n * 400 InvalidAssignableScopePrefix - The Assignable Scope prefix '{0}' must correspond to the current database account '{1}'.\n\n * 409 RoleDefinitionWithSameNameExists - A Role Definition cannot be updated with a name that already exists.\n\n * 400 InvalidDataAction - '{0}' does not match any of the data actions supported by the providers."
+          }
+        }
+      },
+      "delete": {
+        "operationId": "SqlRoleDefinition_Delete",
+        "x-ms-examples": {
+          "CosmosDBSqlRoleDefinitionDelete": {
+            "$ref": "./examples/CosmosDBSqlRoleDefinitionDelete.json"
+          }
+        },
+        "description": "Deletes an exisiting Azure Cosmos DB Role Definition.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/roleDefinitionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Delete is a 202 for Role Assignment from the PDF, but 200 for Role Definition. Is this correct?